invenio_communities/views/communities.py

```diff
@@ -63,28 +66,47 @@ def communities_new():
     )
 
 
+@pass_community
+@pass_community_logo
+def communities_detail(community=None, logo=None, pid_value=None):
```

Comment:
question: this function and the lower `invitations_details` are comming form upstream or from another PR no? they are not needed for the types/organization feature no?